exampleSite/content/_index/table.md

```diff
@@ -3,117 +3,147 @@ fragment = "table"
 weight = 100
 
 title = "Syna is flexible and powerful"
-subtitle = "You can customize your page with our various fragments"
+subtitle = "Customize your page with our various fragments"
 
 [header]
   [[header.values]]
-    text = "Fragment Name"
+    text = "Fragment"
 
   [[header.values]]
     text = "Description"
 
 [[rows]]
   [[rows.values]]
-    text = "Buttons"
+    button = "Buttons"
+    color = "link p-0"
```

Comment:
Why the `p-0`? That is a bit hacky as it works knowing our code, but in my view shouldn't be used or else users might pick it up.

Reply:
`.p-0` is a bootstrap class used for removing any padding. Since we're using `.btn` for button there is extra padding which is not good inside a table for a link. Removing it makes the link not look good.

Reply:
Sounds good.